cpp/oneapi/dal/algo/kmeans/detail/train_init_centroids.hpp

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
```

Comment:
Please start using UXL copyright notice.

```suggestion
* Copyright contributors to the oneDAL project
```